src/main/java/com/openshift/internal/restclient/http/BadRequestException.java

```diff
@@ -1,27 +0,0 @@
-/******************************************************************************* 
- * Copyright (c) 2011 Red Hat, Inc. 
- * Distributed under license by Red Hat, Inc. All rights reserved. 
- * This program is made available under the terms of the 
- * Eclipse Public License v1.0 which accompanies this distribution, 
- * and is available at http://www.eclipse.org/legal/epl-v10.html 
- * 
- * Contributors: 
- * Red Hat, Inc. - initial API and implementation 
- ******************************************************************************/ 
-package com.openshift.internal.restclient.http;
-
-/**
- * @author André Dietisheim
- */
-public class BadRequestException extends HttpClientException {
```

Comment:
We should not be exposing those exception.  I believe I have captured all exceptions and wrappered them in our OpenShiftException or some flavor of that.